concrete/views/dialogs/file/bulk/storage.php

```diff
@@ -1,59 +1,85 @@
-<?php defined('C5_EXECUTE') or die('Access Denied.'); ?>
-<?php
-if (!empty($files)) {
+<?php /** @noinspection PhpComposerExtensionStubsInspection */
 
-    // let's check the storage locations to see if all the files use the same and check the correct checkbox
-    $usedStorageLocations = [];
-    $currentStorageLocation = null;
-    foreach ($files as $file) {
-        $fileStorageLocationID = $file->getStorageLocationID();
-        if (!in_array($fileStorageLocationID, $usedStorageLocations)) {
-            $usedStorageLocations[] = $fileStorageLocationID;
-        }
-    }
-    if (count($usedStorageLocations) === 1) {
-        $currentStorageLocation = $usedStorageLocations[0];
-    } ?>
+defined('C5_EXECUTE') or die('Access Denied.');
+
+use Concrete\Controller\Dialog\File\Bulk\Storage;
+use Concrete\Core\Entity\File\File;
+use Concrete\Core\Entity\File\StorageLocation\StorageLocation as StorageLocationEntity;
+use Concrete\Core\Support\Facade\Application;
+use Concrete\Core\Utility\Service\Identifier;
+
+/** @var Storage $controller */
+/** @var StorageLocationEntity[] $locations */
+/** @var File[] $files */
+$app = Application::getFacadeApplication();
+/** @var Identifier $idHelper */
+$idHelper = $app->make(Identifier::class);
+?>
+
+<?php if (!empty($files)): ?>
+
+    <?php
+        // let's check the storage locations to see if all the files use the same and check the correct checkbox
+        $usedStorageLocations = [];
+        $currentStorageLocation = null;
 
-    <form data-dialog-form="bulk-file-storage" method="post" action="<?= $controller->action('submit'); ?>">
+        foreach ($files as $file) {
+            $fileStorageLocationID = $file->getStorageLocationID();
+            if (!in_array($fileStorageLocationID, $usedStorageLocations)) {
+                $usedStorageLocations[] = $fileStorageLocationID;
+            }
+        }
+        if (count($usedStorageLocations) === 1) {
+            $currentStorageLocation = $usedStorageLocations[0];
+        }
+    ?>
 
-        <?php foreach ($files as $file) { ?>
-            <input type="hidden" name="fID[]" value="<?= $file->getFileID(); ?>" />
-        <?php } ?>
+    <form data-dialog-form="bulk-file-storage" method="post" action="<?php echo $controller->action('submit'); ?>">
+        <?php foreach ($files as $file): ?>
+            <input type="hidden" name="fID[]" value="<?php echo $file->getFileID(); ?>"/>
+        <?php endforeach; ?>
```

Comment:
I don't want this to hold up merging but this isn't syntax that we advocate for use in the core. Can we please clean this up in a subsequent pull request?